app/views/layouts/_footer.html.erb

```diff
@@ -2,21 +2,25 @@
 <div id="footer" role="contentinfo" class="region">
 	<h3 class="landmark heading"><%= ts('Footer', key: 'footer') %></h3>
   <ul class="navigation actions" role="navigation">
-    <% unless !(logged_in? || logged_in_as_admin?) || Skin.in_chooser.empty?%>
-      <li class="module group">
-        <h4 class="heading"><%= ts('Customize', key: 'footer') %></h4>
-        <ul class="menu">
-          <li aria-haspopup="true">
-            <a href="#"><%= ts('Choose Skin', key: 'footer') %></a>
-            <ul id="skin_chooser" class="expandable secondary">
-              <li><%= link_to ts('Default', key: 'footer'), unset_skins_path %></li>
-              <% Skin.in_chooser.order(:title).each do |s| %>
-                <li><%= link_to s.title, set_skin_path(s) %></li>
-              <% end %>
-            </ul> 
+    <% unless !(logged_in? || logged_in_as_admin?) %>
+      <% cache('Skins_menu/v1/'+(Rails.cache.fetch('skins_generation/site_skin') || '0')) do %>
+        <% unless Skin.in_chooser.empty? %>
+          <li class="module group">
+            <h4 class="heading"><%= ts('Customize', key: 'footer') %></h4>
```

Comment:
I have cut and paste what you have above and it says that there is no difference.